src/assets/css/history.css

```diff
@@ -411,3 +398,59 @@ body {
   border: 1px solid hsla(228, 16%, 98%, 0.1);
   opacity: 1;
 }
+
+.context-menu {
+  position: absolute;
+  z-index: 200;
+  top: 20px;
+  left: 20px;
+  background: hsla(120, 2%, 21%, 1);
+  border-radius: 5px;
+  color: #eee;
+  box-shadow: 1px 1px 15px rgba(0, 0, 0, 0.35);
+  display: none;
+}
+
+.context-menu.visible {
+  display: block;
+}
+
+.context-menu ul {
+  list-style: none;
+  margin: 5px 0;
+  padding: 0;
+  width: 120px;
+  height: auto;
+}
+
+.context-menu ul li {
+  display: inline-block;
+  width: 120px;
+  padding: 0 10px;
+  line-height: 30px;
+  font-family: Segoe UI, Roboto, Oxygen, Ubuntu, Fira Sans, Droid Sans,
+    Helvetica Neue, sans-serif;
+  font-size: 13px;
+}
+
+.context-menu ul li svg {
+  width: 12px;
+  height: 12px;
+  display: inline-block;
+  margin: 0 8px;
+}
+
+.context-menu ul li svg path {
+  fill: rgba(240, 240, 240, 0.9);
+}
+
+#context-menu-delete {
+  color: #ff6666;
+}
+
+#context-menu-delete svg path {
+  fill: #ff6666;

```

Comment:
Codacy found an issue: [Expected "#ff6666" to be "#f66" (color-hex-length)](https://app.codacy.com/manual/lenny.glk/Drop/pullRequest?prid=5327605)